src/Cli.Tests/TestHelper.cs

```diff
@@ -895,6 +895,188 @@ public static Process ExecuteDabCommand(string command, string flags)
         }
       }";
 
+        public const string CONFIG_TO_MERGE =
+          @"{" +
+            @"""$schema"": """ + DAB_DRAFT_SCHEMA_TEST_PATH + @"""" + "," +
+            @"""data-source"": {
+          ""database-type"": ""mssql"",
+          ""connection-string"": """",
+          ""options"":{
+            ""set-session-context"": true
+          }
+        },
+        ""runtime"": {
+          ""rest"": {
+            ""path"": ""/api"",
+            ""enabled"": true
+          },
+          ""graphql"": {
+            ""path"": ""/graphql"",
+            ""enabled"": true,
+            ""allow-introspection"": true
+          },
+          ""host"": {
+            ""mode"": ""production"",
+            ""cors"": {
+              ""origins"": [],
+              ""allow-credentials"": false
+            },
+            ""authentication"": {
+              ""provider"": ""StaticWebApps""
+            }
+          }
+        },
+        ""entities"": {
+          ""book"": {
+            ""source"": ""s001.book"",
+            ""permissions"": [
+              {
+                ""role"": ""anonymous"",
+                ""actions"": [
+                  ""*""
+                ]
+              }
+            ]
+          },
+          ""author"": {
+            ""source"": ""s001.authors"",
+            ""permissions"": [
+              {
+                ""role"": ""anonymous"",
+                ""actions"": [
+                  ""*""
+                ]
+              }
+            ]
+          }
+        }
+      }";
+
+        public const string CONFIG_TO_MERGE_WITH =
```

Comment:
>CONFIG_TO_MERGE_WITH [](http://example.com/codeflow?start=28&length=20)

For clarity, could you introduce naming like mentioned in your PR description: base config versus something like 'additionalconfig'